src/joystick/windows/SDL_xinputjoystick.c

```diff
@@ -386,7 +393,7 @@ SDL_XINPUT_JoystickOpen(SDL_Joystick * joystick, JoyStick_DeviceData *joystickde
         return SDL_SetError("Failed to obtain XInput device capabilities. Device disconnected?");
     }
     SDL_zero(state);
-    joystick->hwdata->bXInputHaptic = (XINPUTSETSTATE(userId, &state) == ERROR_SUCCESS);
+    joystick->hwdata->bXInputHaptic = (SDL_bool)(XINPUTSETSTATE(userId, &state) == ERROR_SUCCESS);
```

Comment:
This should be:
`(XINPUTSETSTATE(userId, &state) == ERROR_SUCCESS) ? SDL_TRUE : SDL_FALSE;`

Reply:
Fixed, also fixed a similar pattern in one other place